com.xycz.simple_live.metainfo.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.xycz.simple_live</id>
```

Comment:
`xycz.com` seems to be unrelated.

Reply:
Here `com.xycz.simple_live` is used because this is the package name for both Android and iOS build of Simple Live. I could replace the reverse DNS with something that I am sure belongs to the author, but wouldn't that cause issues internally in Flutter? Keeping this just for being bug-free.

I can test with another reverse DNS name if this is strictly required.

Reply:
> Here com.xycz.simple_live is used because this is the package name for both Android and iOS build of Simple Live.

This contradict 

> The domain used for the application ID is [controlled by the application developer(s)](https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository) and the [application id guidelines](https://docs.flathub.org/docs/for-app-authors/requirements#application-id) are followed.

that you did acknowledge.

Reply:
fixing this is not optional.

Reply:
Done. Tested the app with updated app-id, seems to be working just fine.